internal/controller/instaslice_controller_test.go

```diff
@@ -884,3 +884,23 @@ func TestFirstFitPolicy_SetAllocationDetails(t *testing.T) {
 		})
 	}
 }
+
+func TestGPUSort(t *testing.T) {
```

Comment:
I think this would make more sense to be an under capacity_test.go. We can leave this as a follow up if you don’t want to address now.